core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -314,22 +329,38 @@ public boolean isServing() {
   }
 
   @Override
-  public long getMasterId(String hostname) {
-    MasterInfo existingMaster = mMasters.getFirstByField(HOSTNAME_INDEX, hostname);
+  public List<WrongProperty> getConfErrors() {
+    return mConfigChecker.getConfErrors();
+  }
+
+  @Override
+  public List<WrongProperty> getConfWarns() {
+    return mConfigChecker.getConfWarns();
+  }
+
+  @Override
+  public ServerConfigurationChecker.Status getConfStatus() {
```

Comment:
these three methods are returning 3 parts of the same report. Can we have just one method to return the full report?

Reply:
I will create a ConfigCheckReport that includes configErrors, configWarns and configStatus.